sandbox/minimal-appdir/README.md

```diff
@@ -1,6 +1,6 @@
 # Minimal Next.js App Router example for UploadThing
 
-<a href="https://stackblitz.com/github/pingdotgg/uploadthing/tree/main/examples/minimal-appdir">
+<a href="https://stackblitz.com/github/pingdotgg/uploadthing-community/tree/main/examples/minimal-appdir">
```

Comment:
Wrong link?

Reply:
looks correct to me? Repo is just private rn